src/azure-cli/azure/cli/command_modules/acs/_consts.py

```diff
@@ -43,8 +43,8 @@
 CONST_AVAILABILITY_SET = "AvailabilitySet"
 
 # vm size
-CONST_DEFAULT_NODE_VM_SIZE = "Standard_DS2_v2"
-CONST_DEFAULT_WINDOWS_NODE_VM_SIZE = "Standard_D2s_v3"
+CONST_DEFAULT_NODE_VM_SIZE = "" 
```

Comment:
```suggestion
CONST_DEFAULT_NODE_VM_SIZE = ""
```

please remove the extra space, that would fail the lint/style check

Reply:
sure, already removed.